src/System.Threading.Channels/src/System/Threading/Channels/SingleConsumerUnboundedChannel.cs

```diff
@@ -170,36 +170,33 @@ public override bool TryComplete(Exception error)
             public override bool TryWrite(T item)
             {
                 SingleConsumerUnboundedChannel<T> parent = _parent;
-                while (true) // in case a reader was canceled and we need to try again
```

Comment:
> I believe we should port these changes to the preview1 branch too.

There's no need to port this to preview 1; it's just removing a small amount of dead code.

Reply:
I am not sure I understand this. the changes in this PR supposedly fixing a couple of issues and not just removing a dead code. I am talking about all changes in this PR. do you think it is not important to have these fixes in preview 1.

Reply:
I'm talking about the removal of the while loop I commented on.

Reply:
ok. thanks. what hurts porting it to the other changes? I am not seeing any risk.

Reply:
> any risk

No change has 0 risk. I generally err on the side of porting only what's needed.

That said, I don't feel strongly about it.